doc/docs.md

```diff
@@ -4105,12 +4105,35 @@ fn copy_all(dupes bool) {
 }
 ```
 
+It also possible to use multiline comments with `/* */` (they can also be mixed with single line comments)
```

Comment:
It is also possible to use multiline comments with `/* */` (they can be mixed with single line comments).